plugins/pipelines/latestBuild.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = joi.reach(schema.models.job.base, 'id');
+const nameSchema = joi.reach(schema.models.job.base, 'name');
+const statusSchema = joi.reach(schema.models.build.base, 'status');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/pipelines/{id}/{jobName}/latestBuild',
```

Comment:
since we already have `GET /pipelines/{id}/jobs?archived={boolean}`, wondering if it would be better to have something like 
```
GET /pipelines/{id}/jobs/{jobName}/latestBuild?status={boolean}
```